kernel/kernel-core/src/main/java/io/mosip/kernel/core/util/CryptoUtil.java

```diff
@@ -15,6 +16,16 @@
  * @since 1.0.0
  */
 public class CryptoUtil {
+	
+	private static Encoder urlSafeEncoder;
+	
+	private static Encoder plainEncoder;
+	
+	
+	static {
+		urlSafeEncoder = Base64.getUrlEncoder().withoutPadding();
+		plainEncoder = Base64.getEncoder().withoutPadding();

```

Comment:
For plain encoder, we don't want the padding to be removed, since apache library was not following that.

Reply:
done